packages/components/app/styles/components/code-block/theme.scss

```diff
@@ -0,0 +1,156 @@
+// THEMING
+
+.hds-code-block--theme-dark {
+  // COLORS
+
+  // Descriptive color names:
+
+  --code-block-color-blue: #2d8eff;
```

Comment:
question: should these be prefixed with `hds`?
- so we avoid possible collisions? (but since they're not declared in the `:root` but scoped to the variable, the only way to override them is to technically override the theme, which is something we may want them to do)
- so "consumers" developers will know where they come from when they look at them in the devtools inspector

Or do we expect them to become official design tokens, hence they will be `--token-***`?

Reply:
Prefixed variables with `hds-` in 6c04d5fe3efa33b958e68e3fa49713e271c7abe1. Noting a slight inconsistency here, the SideNav has `hds-`-prefixed variables whereas the AppFooter ones don't have prefixes.